locale/ar/index.md

```diff
@@ -11,8 +11,8 @@ labels:
   tagline-lts: موصى به لجميع المستخدمين
   changelog: سجل التغييرات
   api: API Docs
-  version-schedule-prompt: أو انظر إلى
-  version-schedule-prompt-link-text: أجندة LTS
+  version-schedule-prompt: For information about supported releases, see the
+  version-schedule-prompt-link-text: release schedule
```

Comment:
@nodejs/nodejs-ar Can someone provide translations for these phrases (which will be put together to form a sentence in the page)?